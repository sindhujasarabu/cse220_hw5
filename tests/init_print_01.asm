.data
_id: .word 3126375
_credits: .word 314
_name: .asciiz "Kevin T. McDonnell"

.include "p12_tester.asm"
