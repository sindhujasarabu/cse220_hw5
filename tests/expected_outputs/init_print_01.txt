3126375 314 Kevin T. McDonnell